src/api.js

```diff
@@ -0,0 +1,17 @@
+import { parseId, post, get } from './util.js';
+
+export async function createGame(name) {
+  const { result } = await post('games', { name });
+  const id = parseId(result);
+  return id;
+}
+
+export async function fetchScores(id) {
+  const { result } = await get(`games/${id}/scores/`);
+  return result;
+}
+
+export async function submitScore(id, data) {
+  const { result } = await post(`games/${id}/scores/`, data);
+  return result;
+}
```

Comment:
- [ ] Also please, change these to `es6 arrow functions`.

Reply:
@mikenath223 I don't understand how this is blocking my PR from being approved. It's not part of the general requirement, and also not part of the Microverse best practice.